### DISCUSSION TITLE
submitting a query takes a long time

### DISCUSSION THREAD

**Discussion by KingOfTheNoobYouKnow** [imartinez/privateGPT#426] [Q&A]
Hello! For me submitting a query does not work as it should, it takes a long time to answer for a simple question about 15-20 minutes however I am on a macbook air m1, is there a solution? Thanks in advance

**Comment by quintonm** [comment]
I have the same problem with a macbook m1 pro w/ 32 GB RAM.   Just over 30 minutes with gpt4-x-vicuna-13B.ggml.q8_0.bin. My colleague got a response back in 30 seconds with the same model using an intel macbook pro w/ 32GB RAM.
